code/modules/mob/living/simple_animal/hostile/mining/elites/goliath_broodmother.dm

```diff
@@ -0,0 +1,267 @@
+	var/tturf = get_turf(target)
+	if(!isturf(tturf))
+		return
+	visible_message("<span class='warning'>[src] digs its tentacles under [target]!</span>")
+	new /obj/effect/temp_visual/goliath_tentacle/broodmother/patch(tturf, src)
+
+/mob/living/simple_animal/hostile/asteroid/elite/broodmother/proc/spawn_children(target)
+	ranged_cooldown = world.time + 40 * revive_multiplier()
+	visible_message("<span class='danger'>The ground churns behind [src]!</span>")
+	for(var/i in 1 to 2)
+		if(children_list.len >= 8)
+			return
+		var/mob/living/simple_animal/hostile/asteroid/elite/broodmother_child/newchild = new /mob/living/simple_animal/hostile/asteroid/elite/broodmother_child(loc)
+		newchild.GiveTarget(target)
+		newchild.faction = faction.Copy()
+		visible_message("<span class='danger'>[newchild] appears below [src]!</span>")
+		newchild.mother = src
+		children_list += newchild
+
+/mob/living/simple_animal/hostile/asteroid/elite/broodmother/proc/rage()
+	ranged_cooldown = world.time + 100 * revive_multiplier()
+	//playsound(src,'sound/voice/insane_low_laugh.ogg', 200, 1) find sound
+	visible_message("<span class='warning'>[src] starts picking up speed!</span>")
+	color = "#FF0000"
+	speed = 0
+	move_to_delay = 3
+	addtimer(CALLBACK(src, .proc/reset_rage), 65)
+
+/mob/living/simple_animal/hostile/asteroid/elite/broodmother/proc/reset_rage()
+	color = "#FFFFFF"
+	speed = 2
+	move_to_delay = 5
+
+/mob/living/simple_animal/hostile/asteroid/elite/broodmother/proc/call_children()
+	ranged_cooldown = world.time + 60 * revive_multiplier()
+	visible_message("<span class='warning'>The ground shakes near [src]!</span>")
+	var/list/directions = GLOB.cardinal.Copy() + GLOB.diagonals.Copy()
+	for(var/mob/child in children_list)
+		var/spawndir = pick_n_take(directions)
+		var/turf/T = get_step(src, spawndir)
+		if(T)
+			child.forceMove(T)
+			playsound(src, 'sound/effects/bamf.ogg', 100, 1)
+
+//The goliath's children.  Pretty weak, simple mobs which are able to put a single tentacle under their target when at range.
+/mob/living/simple_animal/hostile/asteroid/elite/broodmother_child
+	name = "baby goliath"
+	desc = "A young goliath recently born from it's mother.  While they hatch from eggs, said eggs are incubated in the mother until they are ready to be born."
+	icon = 'icons/mob/lavaland/lavaland_monsters.dmi'
+	icon_state = "goliath_baby"
+	icon_living = "goliath_baby"
+	icon_aggro = "goliath_baby"
+	icon_dead = "goliath_baby_dead"
+	icon_gib = "syndicate_gib"
+	maxHealth = 30
+	health = 30
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	attacktext = "bashes against"
+	attack_sound = 'sound/weapons/punch1.ogg'
+	throw_message = "does nothing to the rocky hide of the"
+	speed = 2
+	move_to_delay = 5
+	mob_biotypes = MOB_ORGANIC|MOB_BEAST
+	mouse_opacity = MOUSE_OPACITY_ICON
+	deathmessage = "falls to the ground."
+	status_flags = CANPUSH
+	var/mob/living/simple_animal/hostile/asteroid/elite/broodmother/mother = null
```

Comment:
needs to be nulled on destroy, and it needs to remove itself from the mothers child list on destroy.

Reply:
done